arelle/plugin/validate/ESEF_2022/DTS.py

```diff
@@ -203,7 +203,7 @@ def lc3wordAdjust(word: str) -> str:
         #        _("Line items that do not require any dimensional information to tag data MUST be linked to \"Line items not dimensionally qualified\" hypercube in http://www.esma.europa.eu/xbrl/esef/role/esef_role-999999 declared in esef_cor.xsd: concept %(concepts)s."),
         #        modelObject=extLineItemsWithoutHypercube, concepts=", ".join(str(c.qname) for c in extLineItemsWithoutHypercube))
         if extLineItemsNotAnchored:
-            val.modelXbrl.error("ESEF.3.3.1.extensionConceptsNotAnchored",
+            val.modelXbrl.warning("ESEF.3.3.1.extensionConceptsNotAnchored",
```

Comment:
I'm not sure this is the correct change for this. I believe that all concepts should be anchored to wider-narrower. There is just a caveat that text blocks dont necessarily need to be. Thus all concepts need to have an error and text blocks need to have a warning or be filtered out completely.

Reply:
Having read again the RTS, Annex IV, point 7 to 10, the situation seems even more complicated.
A differentiation based on the data type is not appropriate. Following point 9, all extension elements that appear in the presentation linkbase under the following root elements must be anchored:

- ifrs-full:StatementOfFinancialPositionAbstract
- ifrs-full:IncomeStatementAbstract
- ifrs-full:StatementOfComprehensiveIncomeAbstract
- ifrs-full:StatementOfCashFlowsAbstract
- ifrs-full:StatementOfChangesInEquityAbstract
- ifrs-full:NotesAccountingPoliciesAndMandatoryTags

However, as per point 10, subtotals do not need to be anchored.

We'll see what we can do...